app/src/main/java/com/flybits/conciergesample/activity/ZonesModulesConcierge.kt

```diff
@@ -80,27 +82,29 @@ class ZonesModulesConcierge: AppCompatActivity() {
                             Concierge.optOut(applicationContext, object : BasicResultCallback {
                                 override fun onSuccess() {
                                 }
+
                                 override fun onException(exception: FlybitsException) {
                                 }
                             })
                         } else {
                             Concierge.optIn(applicationContext, object : BasicResultCallback {
                                 override fun onSuccess() {
                                 }
+
                                 override fun onException(exception: FlybitsException) {
                                 }
                             })
                         }
                     }
                 }
             } else {
-               // Concierge not connected.
+                // Concierge not connected.
             }
         }
 
         call_handleActionableLink.setOnClickListener {
             val transaction = supportFragmentManager.beginTransaction()
-            Concierge.handleActionableLink(this, Uri.parse("details://?contentId=D68B465C-34B5-41E9-877C-5813CA040C62"))
+            Concierge.handleActionableLink(this, Uri.parse("app://?name=id1"))
```

Comment:
why we have this here?
Aren't we using the content from ES?

Reply:
@caiodias I will change that, I was using to test before the tenant with actionable link.

Reply:
The sample isn't about RequestEvents.
Why add to the code and make things more complex?